### PR TITLE
fix: fix fp with name axel by removing it from rce rule (932260 PL1)

### LINKED ISSUE #3644: FP 932260: Name of "Axel"
### Description

This is very similar to #3490, but with a fill name field. For example, "Axel Smith" gets blocked.

### How to reproduce the misbehavior (-> curl call)

```sh
curl -i -H "x-format-output: txt-matched-rules" 'https://sandbox.coreruleset.org?name=axel+smith'
```

```
Date: Wed, 03 Apr 2024 15:41:16 GMT
Content-Type: text/plain
Transfer-Encoding: chunked
Connection: keep-alive
X-Unique-ID: Zg14nKzdFlh2Fu_y9nB4ewAAAIg
x-backend: apache-nightly

932260 PL1 Remote Command Execution: Direct Unix Command Execution
949110 PL1 Inbound Anomaly Score Exceeded (Total Score: 5)
980170 PL1 Anomaly Scores: (Inbound Scores: blocking=5, detection=5, per_pl=5-0-0-0, threshold=5) - (Outbound Scores: blocking=0, detection=0, per_pl=0-0-0-0, threshold=4) - (SQLI=0, XSS=0, RFI=0, LFI=0, RCE=5, PHPI=0, HTTP=0, SESS=0, COMBINED_SCORE=5)
```

### PR DESCRIPTION
**Pull request by franbuehler**
This PR resolves #3644 by removing the German name `Axel` from RCE rule 932260. It also adds a test to validate the removal of the FP.

### CODE REVIEW

File: regex-assembly/exclude/unix-shell-fps-pl1.ra [hunk 1]
```diff
@@ -120,6 +120,8 @@ awk
 awk@
 aws
 aws@
+axel
+axel@
```
Comment (theseion) [inline regex-assembly/exclude/unix-shell-fps-pl1.ra:124]:
```suggestion
axel@
axel~
```

**Review by theseion — CHANGES_REQUESTED** [review]

**Comment by franbuehler** [comment]
Thank you for the review @theseion.

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml [hunk 2]
```diff
@@ -750,3 +750,21 @@ tests:
...
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+              Host: localhost
+              User-Agent: "OWASP CRS test agent"
+            method: POST
+            port: 80
+            uri: /post
+            version: HTTP/1.0
```
Comment (theseion) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml:766]:
```suggestion
            version: HTTP/1.1
```

**Review by theseion — CHANGES_REQUESTED** [review]
Sorry, missed that.

**Review by theseion — APPROVED** [review]

**Pull request merged**